### ISSUE TITLE
Full Page Screenshot Not Working

### ISSUE THREAD

**Issue by lavonbgrant** [responsively-org/responsively-app#408]
I am trying to take a screenshot for a general laptop of our home page but it is not working. While it goes through the entire process from "taking" to "processing" to "complete" -- no actual screenshot is taken.

I have uninstalled and reinstalled and trying clearing the cache folder. I have also tried moving the screenshot folder to several places. Nothing works. Please advise .

**Comment by jjavierdguezas** [comment]
hi @lavonbgrant
Could you share your environment settings (OS, etc)? (press F1 inside Responsively and copy-paste the info showed)
Do that website has a public URL in order to reproduce with it?
Do this happened only for that website or it happens with other public websites (if so please share)?

**Comment by lavonbgrant** [comment]
Jose,

 

Here’s the information requested:

 

Version: 0.11.0

Electron: 9.1.1

Chrome: 83.0.4103.122

Node.js: 12.14.1

V8: 8.3.110.13-electron.0

OS: Windows_NT x64 10.0.19041

 

The screenshot feature doesn’t work on any site, including https://www.google.com/

 

Thanks!

**Comment by jjavierdguezas** [comment]
Sorry @lavonbgrant  I can't reproduce it, and the fact that it does not work on any site suggests me that it is something specific of your environment.
Any other information you can share will help us
For example:
- Can you share a video/gif of the problem?
- Are you admin in this Windows PC ?
- Did you try to open Responsively as Admin?
- Do you have any antivirus installed? If so, try stopping it first

**Comment by lavonbgrant** [comment]
Jose,




Appreciate the prompt response--will attempt all of your suggestions and see if any work.




Thanks again!




LG




Get Outlook for Android







On Thu, Aug 20, 2020 at 8:59 AM -0500, "José Javier Rodríguez Zas" <notifications@github.com> wrote:















Sorry @lavonbgrant  I can't reproduce it, and the fact that it does not work on any site suggests me that it is something specific of your environment.

Any other information you can share will help us

For example:

Can you share a video/gif of the problem?
Are you admin in this Windows PC ?
Did you try to open Responsively as Admin?
Do you have any antivirus installed? If so, try stopping it first

**Comment by uhlhosting** [comment]
We have somehow same issue, yet I discovered here it is a matter of the page where we take the screenshot, from 4 tested pages, one of them had visible issues, and mainly because portions of the header were left outside.

See image bellow, can be compared to original [site](https://www.tkr-bausysteme.ch)

![Tkr-bausysteme-Full-MacBook-Air](https://user-images.githubusercontent.com/6324047/97836119-e0364280-1cdb-11eb-91ff-e10e33af7a69.jpg)
For some reasons this whole part is missing:
<img width="1678" alt="image" src="https://user-images.githubusercontent.com/6324047/97836261-32776380-1cdc-11eb-8c67-ca22ebcdb7d1.png">

Basically the whole Menu and TopBar Menus are not showing in the printscreen.

Using: 
Version: 0.14.1
Electron: 9.3.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 20.1.0
